firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -71,7 +93,7 @@ public static FirebaseInstallations getInstance(@NonNull FirebaseApp app) {
   @NonNull
   @Override
   public Task<String> getId() {
-    return Tasks.forResult("fid-is-better-than-iid");
+    return Tasks.call(executor, () -> getFirebaseInstallationId());
```

Comment:
this would be more cleanly expressed as a method reference (`this::getFirebaseInstallationId`)